### PR TITLE
cop it up

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: Guardfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```
Comment (michaelwehrley) [inline Guardfile]: String literals will be frozen by default in ruby 3.0

Comment (ericcf) [inline Guardfile]: Yup, until then this "magic" comment adds that to existing code.

File: app/models/smartphone.rb [hunk 2]
```diff
@@ -9,6 +10,6 @@ class Smartphone < ActiveRecord::Base
   before_save :sanitize_number
 
   def sanitize_number
-    self.number = self.number.gsub(/[^0-9]/, "")
+    self.number = number.gsub(/[^0-9]/, "")
   end
 end
```
Comment (michaelwehrley) [inline app/models/smartphone.rb]: how did `rubocop` miss this?! :cop:

File: spec/views/pending/participants/index.html.erb_spec.rb [hunk 3]
```diff
@@ -3,7 +3,7 @@
 require "spec_helper"
```
Comment (michaelwehrley) [inline spec/views/pending/participants/index.html.erb_spec.rb]: you forgot `# frozen_string_literal: true`

**Comment by michaelwehrley** [comment]
looks good!

**Pull request merged**